vowpalwabbit/cost_sensitive.cc

```diff
@@ -147,7 +154,7 @@ void parse_label(parser* p, shared_data* sd, void* v, v_array<VW::string_view>&
         else
         {
           wclass f = {-FLT_MAX, 0, 0., 0.};
-          ld->costs.push_back(f);
+          ld.costs.push_back(f);
```

Comment:
would using emplace_back save us a copy? here and in the other push_back

Reply:
emplace_back uses the input parameters as constructor params, there's no constructor for that object right now; but we should totally make that change when we can